### PR TITLE
*: option capacity, 0 for disk capacity

### PR DESCRIPTION
**Pull request by overvenus**
PTAL @siddontang @BusyJay  @huachaohuang

### CODE REVIEW

**Comment by BusyJay** [comment]
Should update store.rs too.

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -443,7 +443,17 @@ fn build_cfg(matches: &Matches, config: &toml::Value, cluster_id: u64, addr: Str
         .or_else(|| get_toml_int_opt(config, "server.capacity"));
     if let Some(cap) = capacity {
         assert!(cap >= 0);
-        cfg.raft_store.capacity = cap as u64;
+        if cap == 0 {
```
Comment (siddontang) [inline src/bin/tikv-server.rs]: we can handle cap == 0 in store heartbeat.

**Comment by BusyJay** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Comment by ngaut** [comment]
Please fix CI

**Comment by andelf** [comment]
Why not use available_space?

**Comment by overvenus** [comment]
@andelf Because it is **capacity**, also there is a field named as [**available**](https://github.com/pingcap/kvproto/blob/b66543960fb3508671152f6c9f4a09ee9d7a2322/proto/pdpb.proto#L201).

**Comment by siddontang** [comment]
LGTM

**Pull request merged**